ocs_ci/ocs/machine.py

```diff
@@ -0,0 +1,127 @@
+import re
+import logging
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs import constants
+
+
+log = logging.getLogger(__name__)
+
+
+def get_machine_objs(machine_names=None):
+    """
+    Get machine objects by machine names
+
+    Args:
+        machine_names (list): The machine names to get their objects
+                              If None, will return all cluster machines
+
+    Returns:
+        list: Cluster machine OCP objects
+    """
+    machines_obj = OCP(
+        kind='Machine', namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE
+    )
+    machine_dicts = machines_obj.get()['items']
+    if not machine_names:
+        return [OCS(**machines_obj) for machines_obj in machine_dicts]
```

Comment:
Not sure I understand this line.
If you want to initialize ocs object for each item in machine_dicts, why not using:
[OCS(**obj) for obj in machine_dicts]
?

Reply:
Done